war/src/main/webapp/WEB-INF/security/SecurityFilters.groovy

```diff
@@ -33,15 +33,13 @@ import hudson.security.ChainedServletFilter
 import hudson.security.UnwrapSecurityExceptionFilter
 import hudson.security.HudsonAuthenticationEntryPoint
 import jenkins.security.BasicHeaderProcessor
-import org.acegisecurity.providers.anonymous.AnonymousProcessingFilter
 import jenkins.security.ExceptionTranslationFilter
-import org.acegisecurity.ui.basicauth.BasicProcessingFilter
+import org.acegisecurity.providers.anonymous.AnonymousProcessingFilter
 import org.acegisecurity.ui.basicauth.BasicProcessingFilterEntryPoint
 import org.acegisecurity.ui.rememberme.RememberMeProcessingFilter
+
+/* required by a bean/hudson.util.spring.BeanBuilder: */
```

Comment:
As this is put directly in the script, I do not think this is necessary (especially a comment inside the import, the probability to be read is very low)

If you do not mind removing this, I would be happy 🥇

Reply:
Removed